classes/game.rb

```diff
@@ -0,0 +1,16 @@
+require_relative 'item'
+
+class Game
+  attr_accessor :title, :platform, :last_played_at
+
+  def initialize(title, platform, last_played_at)
+    @title = title
+    @platform = platform
```

Comment:
The UML diagram doesn't specify a **platform** property. I think this should be **multiplayer**
![image](https://github.com/henokkhm/microverse-module-4-ruby-group-capstone/assets/110997325/5496d57d-e2a6-49b7-a1e8-5fcb6b192706)